test/common/upstream/load_balancer_fuzz.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+
+package test.common.upstream;
+
+import "validate/validate.proto";
+import "envoy/config/cluster/v3/cluster.proto";
+import "google/protobuf/empty.proto";
+
+message UpdateHealthFlags {
+  //The boolean corresponds to whether this action updates the failover set or not
```

Comment:
Nit: please add a space between `//` and the text.

Reply:
Added